src/components/Heading/Heading.tsx

```diff
@@ -85,19 +20,23 @@ type HeadingProps = {
   /**
    * Prop to set the desired typography value used in design. Acceptable values
    * match those used across the design system.
+   *
+   * For details, see https://chanzuckerberg.github.io/edu-design-system/?path=/story/design-tokens-tier-2-usage--typography
    */
   preset?: Preset;
-  /**
-   * This prop is **deprecated**.
-   * @deprecated
-   */
-  size: HeadingSize;
-  /**
-   * This prop is **deprecated**.
-   * @deprecated
-   */
-  variant?: Variant;
-} & React.HTMLAttributes<HTMLHeadingElement>;
+};
+
+/**
+ * Given a certain HeadingElement, what is the default preset to use?
+ */
+const headingPresetMap: Record<HeadingElement, Preset> = {
+  h1: 'headline-lg',
```

Comment:
Not blocking idea:

When using the component, it feels like I'd rather write this:

```jsx
<Heading as="h1" preset="h2" />
```

so that I'm describing a preset relative to other headings.

Reply:
✅ I think I follow and that could work. The intent was to allow `preset` to always match the typography name from design to start, so that engineers don't have to reason about which level means what preset. 

I plan to host a workshop for the migration to 14, and can see how people think thru what to use